src/lib/components/landing/Stat.svelte

```diff
@@ -21,24 +24,24 @@
   <div class="w-2/4">
     {@html description}
   </div>
-  <div
-    class="grid grid-cols-{$stats.length} grid-flow-col justify-center p-4 my-4 gap-y-9 {$stats.length >
-    4
-      ? 'w-full'
-      : 'w-1/2'}"
-  >
+  <div class="grid grid-cols-{$stats.length} grid-flow-col justify-center p-4 my-4 gap-y-9 {width}">
     {#each $stats as stat (`${authString}-${stat.key}-${stat.label}`)}
       <div class="p-4 not-last:border-r border-surface-500">
         <div
           data-testid="value-{authString}-{stat.key}-{stat.label}"
           class="flex flex-col justify-center items-center text-2xl"
         >
-          {#await stat.value}
+          {#await Promise.allSettled(promiseList(stat))}
```

Comment:
This causes the page to wait for all stats to finish loading, rather then letting each one finish in its own time.
If this need to happen this way I think we should either show a single spinner.